libs/design-system/navigation/header/src/lib/mobile-menu/mobile-menu.component.scss

```diff
@@ -1,23 +1,24 @@
 @use '../../../../../styles/utils';
+@use '../../../../../styles/vars' as vars;
 
 :host {
   display: block;
   width: 100%;
-  background-color: var(--mat-sys-surface-dim);
+  background-color: vars.$surface-dim;
 
   $header-height: 4rem;
 
-  --mat-expansion-container-shape: var(--mat-sys-corner-none);
-  --mat-expansion-container-background-color: var(--mat-sys-surface-dim);
+  --mat-expansion-container-shape: vars.$corner-none;
```

Comment:
Missing interpolation in this file: `#{...}`, though this should probably be updated to use an overrides mixin instead...